### PR TITLE
General cleanup, better replies, more regex paterns

### PR DESCRIPTION
**Pull request by ghost**
Yes I am shit at RegEx, just fix the changes as everything I changed is probably wrong.

### CODE REVIEW

File: replies.go [hunk 1]
```diff
@@ -165,14 +165,14 @@ var replies = []Reply{
...
+		message: "Impact for older versions of Minecraft (pre 1.8) is no longer availible to comply with Mojang's EULA.",
 	},
 	{
 		pattern: `(modpack|\bftb\b|rlcraft|skyfactory|valhelsia|pixelmon|sevtech)`,
 		message: "Impact is generally incompatible with modpacks and support will not be provided if you encounter bugs with them. It's likely your game will just crash on startup.",
 	},
 	{
-		pattern: `good bot`,
-		message: "thank you *nuzzles*",
+		pattern: `((good)\s*(bots?|human))`,
```
Comment (ghost) [inline replies.go]: This is my main concern, there is probably better RegEx for this.

File: replies.go [hunk 2]
```diff
@@ -121,12 +121,12 @@ var replies = []Reply{
...
 		excludeRoles: []Role{Donator},
 	},
 	{
 		pattern:      `(1\.15.*?(fucking|get|where|need|asap|update|coming|support|release|impact|version|eta|when|out|support)|(fucking|get|where|need|asap|update|coming|support|release|impact|version|eta|when|out|support).*?1\.15)`,
-		message:      "1.15.2 has been released! Download the newest installer [here](https://impactclient.net/?brady-money-grubbing-completed=true).",
+		message:      "The current version of Impact is Minecraft release 1.15.2. Download the installer [here](https://impactclient.net/?brady-money-grubbing-completed=true).",
```
Comment (ghost) [inline replies.go:129]: Impact 1.15 has been out for a while now, so made it sound a little less "New!".

File: replies.go [hunk 3]
```diff
@@ -71,15 +71,15 @@ var replies = []Reply{
...
+		message: "To use LiteLoader, [please follow this tutorial](https://github.com/ImpactDevelopment/ImpactIssues/wiki/Adding-LiteLoader).",
 	},
 	{
 		pattern: `(web\s?)?(site|page)`,
 		message: "[Impact Website](https://impactclient.net)",
 	},
 	{
 		pattern: `issue|bug|crash|error|suggest(ion)?s?|feature|enhancement`,
-		message: "Use the [GitHub repo](https://github.com/ImpactDevelopment/ImpactIssues/issues) to report issues/suggestions!",
+		message: "If you have encountered a bug whilst using Impact that cannot be answered in the help channels, or have a suggestion for a feature, use the [GitHub repo](https://github.com/ImpactDevelopment/ImpactIssues/issues) to report issues/suggestions!",
```
Comment (ghost) [inline replies.go:82]: may be bad addition, may not be

File: replies.go (hunk 1 above)
Comment (Fakeaccount12312) [reply replies.go]: This is a tiny bit shorter:
```suggestion
		pattern: `(good\s*(bot|human))`,
```

**Pull request closed without merging**